include/ensmallen_bits/problems/nsga2_test_sch_function.hpp

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file nsga2_test_sch_function.hpp
+ * @author Sayan Goswami
+ *
+ * ensmallen is free software; you may redistribute it and/or modify it under
+ * the terms of the 3-clause BSD license.  You should have received a copy of
+ * the 3-clause BSD license along with ensmallen.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef ENSMALLEN_PROBLEMS_NSGA2_TEST_SCH_FUNCTION_HPP
+#define ENSMALLEN_PROBLEMS_NSGA2_TEST_SCH_FUNCTION_HPP
+
+namespace ens {
+namespace test {
+
+template<typename MatType = arma::mat>
+class NSGA2TestFuncSCH {
+ public:
+  NSGA2TestFuncSCH() : numObjectives(2), numVariables(1)
+  {/* Nothing to do here. */}
+
+  std::vector<double> Evaluate(const MatType& coords) {
```

Comment:
I think we should use `arma::vec` here, since that is the choice that is normally made in the library

Reply:
Sure, will change it!